pysatNASA/instruments/timed_guvi.py

```diff
@@ -0,0 +1,233 @@
+
+    return
+
+
+# ----------------------------------------------------------------------------
+# Instrument functions
+#
+# Use the default CDAWeb and pysat methods
+
+# Set the list_files routine
+fname = 'TIMED_GUVI_L1C-2-disk-IMG_{year:04d}{day:03d}'\
+        '{hour:02d}{minute:02d}{second:02d}-{end:13d}_'\
+        'REV{rev:06d}_Av{version:02d}-{versionb:02d}r{revision:03d}.nc'
+supported_tags = {'': {'': fname}}
+list_files = functools.partial(mm_gen.list_files,
+                               supported_tags=supported_tags,)
+
+# Set the download routine
+basic_tag = {'remote_dir': ''.join(('/pub/data/timed/guvi/levels_v13/level1c/imaging',
+                                    '/{year:4d}/{day:03d}/')),
+             'fname': fname}
+download_tags = {'': {'': basic_tag}}
+download = functools.partial(cdw.download, supported_tags=download_tags)
+
+# Set the list_remote_files routine
+list_remote_files = functools.partial(cdw.list_remote_files, supported_tags=download_tags)
+
+# Set the load routine
+def load(fnames, tag=None, inst_id=None):
+    """Load TIMED GUVI data into `xarray.DataSet` and `pysat.Meta` objects.
+ 
+    This routine is called as needed by pysat. It is not intended
+    for direct user interaction.
+ 
+    Parameters
+    ----------
+    fnames : array-like
+        iterable of filename strings, full path, to data files to be loaded.
+        This input is nominally provided by pysat itself.
+    tag : string
+        tag name used to identify particular data set to be loaded.
+        This input is nominally provided by pysat itself.
+    inst_id : string
+        Satellite ID used to identify particular data set to be loaded.
+        This input is nominally provided by pysat itself.
+    
+    Returns
+    -------
+    data : xr.DataSet
+        A xarray DataSet with data prepared for the pysat.Instrument
+    meta : pysat.Meta
+        Metadata formatted for a pysat.Instrument object.
+ 
+    Note
+    ----
+    Any additional keyword arguments passed to pysat.Instrument
+    upon instantiation are passed along to this routine.
+ 
+    Examples
+    --------
+    ::
+ 
+        inst = pysat.Instrument('timed', 'guvi')
+        inst.load(2005, 171)
+ 
+    """
+    labels = {  'units': ('units', str), 'name': ('long_name', str),
+                'notes': ('notes', str), 'desc': ('desc', str),
+                'plot': ('plot_label', str), 'axis': ('axis', str),
+                'scale': ('scale', str),
+                'min_val': ('value_min', float),
+                'max_val': ('value_max', float),
+                'fill_val': ('fill', float)}
+    meta = pysat.Meta(labels=labels)
+
+    daysubset,nightsubset,auroralsubset=None,None,None
+    for path in fnames:
+        dataset=xr.load_dataset(path)
+        
+        #separate into auroral, night and day datasets
+        auroralkeys=list(filter(lambda k:"_AURORAL" in k[-8:],dataset.keys()))
```

Comment:
I'm not exactly sure on what the spacing is for lamdas, I haven't been using them. 

Spaces are required around ' = '

```suggestion
        auroralkeys = list(filter(lambda k: "_AURORAL" in k[-8:], dataset.keys()))
```